tests/Aspire.Hosting.Redis.Tests/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+  <!-- NOTE: This line is only required because we are using P2P references, not NuGet. It will not exist in real apps. -->
+  <Import Project="../../src/Aspire.Hosting.AppHost/build/Aspire.Hosting.AppHost.props" />
```

Comment:
Are these Build props/targets files really required? It would be nice if we didn't have to have them in every single hosting test project.

Reply:
Didn't need it for my live tests for the WithDockerfile changes:

https://github.com/dotnet/aspire/blob/main/tests/Aspire.Hosting.Containers.Tests/Aspire.Hosting.Containers.Tests.csproj

Reply:
I changed it locally to remove all these imports and in the end it looks like @mitchdenny 's Containters.Tests project. One thing that changed too is 

```
var volumeAnnotation = redis.Resource.Annotations.OfType<ContainerMountAnnotation>().Single();
Assert.Equal("Aspire.Hosting.Tests-myRedis-data", volumeAnnotation.Source);
```

Where in @davidfowl 's branch `TestDistributedApplicationBuilder` would come from an `Include` and the test would check for `"Aspire.Hosting.Redis.Tests-myRedis-data"`. I believe the include was not really required for this PR and this doesn't matter, but please confirm in case I missed something.